decidim-comments/app/frontend/comments/comment.component.jsx

```diff
@@ -51,11 +51,12 @@ class Comment extends Component {
             { body }
           </p>
         </div>
-        {this._renderReplies()}
         <div className="comment__footer">
```

Comment:
Are you sure this is the actual layout being used in the last decidim-design version?